ColorPicker/Views/AuxViews.swift

```diff
@@ -0,0 +1,84 @@
+import SwiftUI
+
+
+struct TitledColorSlider: View {
+  let title: String
+  let tint: Color
+  @Binding var sliderValue: Double
+  
+  var body: some View {
+    VStack {
+      Text(title)
+        .font(.title2)
+      HStack {
+        Slider(value: $sliderValue)
+          .tint(tint)
+        Text("\(Int(sliderValue * 255))")
+      }
+    }
+  }
+}
+
+struct ThreeSliders: View {
+  @Binding var redColor: Double
+  @Binding var greenColor: Double
+  @Binding var blueColor: Double
+  
+  var body: some View {
+    VStack {
+      TitledColorSlider(title: "Red", tint: .red, sliderValue: $redColor)
+      TitledColorSlider(title: "Green", tint: .green, sliderValue: $greenColor)
+      TitledColorSlider(title: "Blue", tint: .blue, sliderValue: $blueColor)
+    }
+  }
+}
+
+struct Title: View {
+  var body: some View {
+    Text("Color Picker")
+      .bold()
+      .font(.largeTitle)
+  }
+}
+
+struct SetColorButton: View {
+  @Binding var foregroundColor: Color
+  @Binding var redColor: Double
+  @Binding var greenColor: Double
+  @Binding var blueColor: Double
+  
+  var body: some View {
+    Button("Set Color") {
+      foregroundColor = Color(red: redColor, green: greenColor, blue: blueColor)
+    }
+      .padding(20)
+      .background(
+        RoundedRectangle(cornerRadius: 16)
+          .fill(.blue)
+          .border(.white, width: 2)
+      )
+      .foregroundColor(.white)
```

Comment:
Look at the view in Dark mode (use Canvas Device Settings and enable dark mode) and you will see that the white border does not have rounded corners. One way to solve is to use an `.overlay` and a `.clipShape`. Here are some modifiers you can use to achieve the rounded corners and rounded borders:
```
.padding(20)
.background(.blue).clipShape(RoundedRectangle(cornerRadius: 16.0))
.overlay(RoundedRectangle(cornerRadius: 16.0).stroke(Color.white, lineWidth: 2))
.foregroundColor(.white)
```
This will round the corners of the background and then overaly another rectangle which only has a white outline.